app/controllers/poros/food.rb

```diff
@@ -0,0 +1,12 @@
+class Food
```

Comment:
It will be hard for Rails to find this class when poros is in controllers. If you bring the poros directory into the app directory, you shouldn't run into the errors you were seeing